docs/developer/test-utilities.rst

```diff
@@ -179,6 +179,35 @@ This mixin provides the core Selenium setup logic and reusable test
 methods that must be used across all OpenWISP modules based on Django to
 enforce best practices and avoid flaky tests.
 
+It also provides a retry mechanism for flaky tests, which executes the a
+failing test multiple times to determine if it is a transient failure.
+This functionality can be configured using the following class attributes:
+
+- ``retry_max``: Number of times to retry a test (default: ``5``).
+- ``retry_delay``: Numbed of seconds to wait before retrying a test
+  (default: ``2``).
```

Comment:
Why do we need to wait to repeat tests? I'd set this to zero to avoid increasing build times.